include/nitro/broken_options/option/toggle.hpp

```diff
@@ -84,7 +117,15 @@ namespace broken_options
                 *ref_ = true;
             }
 
-            ++given_;
+            if (arg.has_prefix())
+            {
+                given_ = 0;
+            }
+            else
+            {
+                ++given_;
+            }
+            //++given_;
```

Comment:
Don't leave unused code around as comments. If you need it later, just use the git history.